addons/payment/models/payment_acquirer.py

```diff
@@ -168,21 +179,19 @@ def _get_default_view_template_id(self):
         ('s2s','Payment from Odoo')],
         default='form', required=True, string='Payment Flow',
         help="""Note: Subscriptions does not take this field in account, it uses server to server by default.""")
-    inbound_payment_method_ids = fields.Many2many('account.payment.method', related='journal_id.inbound_payment_method_ids', readonly=False)
+    inbound_payment_method_id = fields.Many2one(
+        comodel_name='account.payment.method',
+        copy=False,
+        default=_get_default_inbound_payment_method_id,
+        help="Used to store the payment method linked to this particular payment acquirer.",
+    )
```

Comment:
This should be a computed (non stored?) field
Add a constraint that checks the `journal_id` has a method for this acquirer too